.gitignore

```diff
@@ -23,3 +23,5 @@ run
 
 # Files from Forge MDK
 forge*changelog.txt
+.vscode/settings.json
+.vscode/launch.json
```

Comment:
Pretty sure this can be simplified to just `.vscode` and it will ignore the whole folder.